include/simdjson/portability.h

```diff
@@ -37,24 +37,28 @@
 
 #if defined(__x86_64__) || defined(_M_AMD64)
 #define SIMDJSON_IS_X86_64 1
-#endif
-#if defined(__aarch64__) || defined(_M_ARM64)
+#elif defined(__aarch64__) || defined(_M_ARM64)
```

Comment:
We should consider changing our nested if statements to use indentation at this point. I've seen a number of projects do it like this:

```c++
#if A
#  if B
#    define ...
#  else
#    define ...
#  endif
#elif C
#  define ...
#else
#  define ...
#endif
```

It seems the reason for this is -- was? -- that the Visual Studio editor will always move # to column 1 when you type it. But I could do either.

Not a change you need to make, but we're definitely increasing the nesting and it's going to be hard to read soon :)

Reply:
I bet that clang-format can do this.

Reply:
with clang-format `IndentPPDirectives: AfterHash`...

```
// clang-format off
#if A
#if B
#define b
#else
#if D
#define a
#endif
#endif
#elif C
#define c
#else
#define d
#endif
// clang-format on
#if A
#  if B
#    define b
#  else
#    if D
#      define a
#    endif
#  endif
#elif C
#  define c
#else
#  define d
#endif
```

Reply:
Right. We have a .clang-format file. If it does not cover IndentPPDirectives, we can add it there.

My belief is that formatting is best handled by tools.